spec/parser_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'spec_helper'
+require "spec_helper"
```

Comment:
how this correspond with

    --require spec_helper

https://github.com/PurchasingPlatform/cxml/pull/1/files#diff-f5deb99143d9a1be756e16d12073da6eR3